test/document_stores/test_elasticsearch.py

```diff
@@ -376,6 +382,22 @@ def test_write_documents_req_for_each_batch(self, mocked_document_store, documen
             mocked_document_store.write_documents(documents)
             assert mocked_bulk.call_count == 5
 
+    @pytest.mark.unit
+    def test_get_vector_similarity_query(self, mocked_document_store):
+        vec_sim_query = mocked_document_store._get_vector_similarity_query(np.random.rand(3).astype(np.float32), 10)
```

Comment:
Is this some smoke test? What's the motivation behind it? Perhaps a comment or two would help a lot :-)

Reply:
I added some doc strings to the new tests.